geopandas/explore.py

```diff
@@ -0,0 +1,896 @@
+        ``True`` includes all columns. ``False`` removes tooltip. Pass string or list of
+        strings to specify a column(s). Integer specifies first n columns to be
+        included. Defaults to ``True``.
+    popup : bool, str, int, list (default False)
+        Input GeoDataFrame attributes for object displayed when clicking.
+        ``True`` includes all columns. ``False`` removes popup. Pass string or list of
+        strings to specify a column(s). Integer specifies first n columns to be
+        included. Defaults to ``False``.
+    highlight : bool (default True)
+        Enable highlight functionality when hovering over a geometry.
+    categorical : bool (default False)
+        If ``False``, ``cmap`` will reflect numerical values of the
+        column being plotted. For non-numerical columns, this
+        will be set to True.
+    legend : bool (default True)
+        Plot a legend in choropleth plots.
+        Ignored if no ``column`` is given.
+    scheme : str (default None)
+        Name of a choropleth classification scheme (requires ``mapclassify`` >= 2.4.0).
+        A :func:`mapclassify.classify` will be used
+        under the hood. Supported are all schemes provided by ``mapclassify`` (e.g.
+        ``'BoxPlot'``, ``'EqualInterval'``, ``'FisherJenks'``, ``'FisherJenksSampled'``,
+        ``'HeadTailBreaks'``, ``'JenksCaspall'``, ``'JenksCaspallForced'``,
+        ``'JenksCaspallSampled'``, ``'MaxP'``, ``'MaximumBreaks'``,
+        ``'NaturalBreaks'``, ``'Quantiles'``, ``'Percentiles'``, ``'StdMean'``,
+        ``'UserDefined'``). Arguments can be passed in ``classification_kwds``.
+    k : int (default 5)
+        Number of classes
+    vmin : None or float (default None)
+        Minimum value of ``cmap``. If ``None``, the minimum data value
+        in the column to be plotted is used. Cannot be higher than minimum data value.
+    vmax : None or float (default None)
+        Maximum value of ``cmap``. If ``None``, the maximum data value
+        in the column to be plotted is used. Cannot be lower than maximum data value.
+    width : pixel int or percentage string (default: '100%')
+        Width of the folium :class:`~folium.folium.Map`. If the argument
+        m is given explicitly, width is ignored.
+    height : pixel int or percentage string (default: '100%')
+        Height of the folium :class:`~folium.folium.Map`. If the argument
+        m is given explicitly, height is ignored.
+    categories : list-like
+        Ordered list-like object of categories to be used for categorical plot.
+    classification_kwds : dict (default None)
+        Keyword arguments to pass to mapclassify
+    control_scale : bool, (default True)
+        Whether to add a control scale on the map.
+    marker_type : str, folium.Circle, folium.CircleMarker, folium.Marker (default None)
+        Allowed string options are ('marker', 'circle', 'circle_marker'). Defaults to
```

Comment:
If we want to change it I would do `circle_marker` not `circle`. The former is based on pixels so it is always visible. The latter on physical units so needs to be tweaked manually.

Reply:
Ah yes, it would be circle_marker then. But what would you think about changing it? The markers might be a bit more "typical" leaflet, but I personally don't find them very useful in practice in most cases.

Reply:
ipyleaflet defaults to markers, GeoViews to dots (aka small circle_marker). I am not a big fan of default circle_marker since it has 10px radius and no fill which makes it large and it is not immediately clear that it is a point. We could do a small filled circle_marker like this:

![Screenshot 2021-09-19 at 11 25 15](https://user-images.githubusercontent.com/36797143/133924018-0f1a7a4a-4ea9-4357-8633-031724267616.png)

Using marker on the same sample is just useless.
![Screenshot 2021-09-19 at 11 25 36](https://user-images.githubusercontent.com/36797143/133924022-02503711-7f7b-4261-92b0-bcc20b6e21ed.png)

I'll change it.